meta_capstone/ViewControllers/InGameViewController.m

```diff
@@ -78,10 +80,13 @@ - (void)viewDidLoad {
     
     //if user is not the host, remove check button
     if ([self.game[@"hostID"] isEqualToString:self.currUser[@"fbID"]]) {
+        self.navigationItem.rightBarButtonItem = nil;
```

Comment:
why add this when the next line overrides it?

Reply:
I couldn't find a way to remove a specific barbuttonitem, so in order to remove the request host button (or check button) for the host (or non-host), i had to set the button to nil to clear it. assigning rightbarbuttonitem doesn't overwrite the previous button, it just adds a button to the bar.